lib/Admin.php

```diff
@@ -612,7 +612,14 @@ public static function get_widen_url_path( $url ) {
 		// Remove the WordPress uploads base path from the URL.
 		if ( is_multisite() ) {
 			$blog_id = get_current_blog_id();
-			$path    = preg_replace( "/^\/wp-content\/uploads\/sites\/$blog_id\//", '', $url_path );
+
+			if ( 1 === $blog_id ) {
```

Comment:
Do we want to add in a number conversion to make sure $blog_id is a number, and not possibly a string?

Example: 
if ( 1 === (int)$blog_id) {

Thoughts?

Reply:
Thanks for the feedback @jdashley27.

In this case we shouldn't need to since `get_current_blog_id()` is a WordPress core function that returns an integer.

See https://developer.wordpress.org/reference/functions/get_current_blog_id/